build-script.sh

```diff
@@ -33,7 +33,7 @@ for dir in $DIRS; do
   if [ $dir == "." ]; then
     rm -rf v3/
     rm -rf v4/
-  elif [[ $dir =~ "v3" ]]; then
+  elif [[ $dir =~ "v3/" ]]; then
```

Comment:
This change is required for the `v4/integrations/nrecho-v3` tests.

The problem here was that the v3 at the end of the package name was matching this if statement unexpectedly. The solution is to throw a slash at the end to ensure that we are talking about a v3 directory and not a v3 module name.